js/ui/windowManager.js

```diff
@@ -617,6 +620,8 @@ const WindowManager = new Lang.Class({
                                         function () { Main.layoutManager.emit('background-clicked'); });
         }));
 
+        this._codingManager = new CodingManager();
```

Comment:
Moved to a separate file and added the "enable-behind-the-screen" GSetting to toggle the feature on and off.